website/content/api-docs/index.mdx

```diff
@@ -90,14 +86,21 @@ $ curl \
 
 ## API Operations
 
-With few documented exceptions, all request body data and response data from
-Vault is via JSON. Vault will set the `Content-Type` header appropriately but
-does not require that clients set it.
+All request data, body and response data to and from Vault is JSON except in a 
+few exceptional cases that are documented. Vault sets the `Content-Type` header 
+appropriately with its response and does not require it from the clients 
+request.
+
+Authentication and Secret engines, that are also referred to as backends or 
+mounts, differ in their implementation of APIs according to functionality and 
+how they're authored.
```

Comment:
This sentence is hard to parse. I am not sure what it is  trying to be  conveyed by this statement. Are you trying to say that each API may have a unique implementation/design?